myuw/templates/handlebars/card/seattle_application.html

```diff
@@ -0,0 +1,116 @@
+    </h3>
+
+    <div class="myuw-card-section last">
+		<h4 class="applicant-card myuw-header-msg"> For application status, contact The Office of the University Registrar</h4>
+			<div class="row">
+                <div class="col-sm-3">
+                    <div class="myuw-card-row-heading">Email</div>
+                </div>
+                <div class="col-sm-9">
+                    <div class="myuw-card-row-content"><a href="mailto:regoff@uw.edu">regoff@uw.edu</a></div>
+                </div>
+            </div> 
+			<div class="row">
+                <div class="col-sm-3">
+                    <div class="myuw-card-row-heading">Phone</div>
+                </div>
+                <div class="col-sm-9">
+                    <div class="myuw-card-row-content">(206) 543-4000</div>
+                </div>
+            </div>
+            <div class="row">
+                <div class="col-sm-3">
+                    <div class="myuw-card-row-heading">Fax</div>
+                </div>
+                <div class="col-sm-9">
+                    <div class="myuw-card-row-content">(206) 616-3222</div>
+                </div>
+            </div>
+            <div class="row">
+                <div class="col-sm-3">
+                    <div class="myuw-card-row-heading">Location</div>
+                </div>
+                <div class="col-sm-9">
+                    <div class="myuw-card-row-content">225 Schmitz Hall</div>
+                </div>
+            </div>
+            <div class="row last">
+                <div class="col-sm-3">
+                    <div class="myuw-card-row-heading">Mail</div>
+                </div>
+                <div class="col-sm-9">
+                    <div class="myuw-card-row-content">Registration and Transcripts Office<br>University of Washington <br>Box 355850 <br>Seattle, WA 98195-5850</div>
+                </div>
+            </div>			
+	</div>
+	<div class="myuw-card-section">	
+
+		<h4 class="applicant-card"> Resources for Seattle Applicants</h4>
+			<ul class="unstyled-list list-leading">
+			    <li><a href="https://www.washington.edu/financialaid/" target="_blank">Learn all about student financial aid and scholarships</a></li>
+			    <li><a href="http://www.washington.edu/students/" target="_blank">Check out the Student guide for academics, student life, and more</a></li>
+			   	<li><a href="http://www.washington.edu/students/reg/calendar.html" target="_blank">View the UW Academic calendars</a></li>
+			</ul>
+
+
+    </div>
+
+{{else}}   
+    <h3 class="myuw-card-title">
+        Your Seattle Application for {{quarter}} {{year}}
+    </h3>
+
+    <div class="myuw-card-section">
+        
+        <div class="clearfix">
+             <span class="btn btn-default card-action-btn myuw-btn-big"><a href="https://sdb.admin.uw.edu/admissions/uwnetid/appstatus.asp"  data-linklabel="Your application status details"><h4 class="applicant-card">View your {{type}} application status</h4></a></span>
```

Comment:
The document outline will read h4 for the SR users.

Reply:
Hmm, I think putting a `<h4 class="sr-only">Heading text</h4>` above the button makes more sense if that is the goal. I'm not sure you need one though since the card title itself should be read as part of the document outline and the button is directly after that.

The heading here as a link is not semantically correct because you are saying "View your Fall application status" is the heading for the content below, but the content below also has a heading for the "Resources".